docs/AddingColorContrastTests.md

```diff
@@ -0,0 +1,22 @@
+# Color Contrast Tests
```

Comment:
># [](start = 0, length = 1)

GitHub has accessibility issues with headers. We've gotten around this by making all headers start with H2